### ISSUE TITLE
Fix the pylint issue: inconsistent-return-statements

### ISSUE THREAD

**Issue by troydai** [Azure/azure-cli#5385]
https://github.com/Azure/azure-cli/blob/a2ec550db69eae478d19e3fe7a29150bc4fef8a0/src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/repository.py#L82

**Comment by djyou** [comment]
Add a disable?

**Comment by djyou** [comment]
It seems to be fixed in this PR https://github.com/Azure/azure-cli/pull/5382

**Comment by troydai** [comment]
No. The disable is added temporarily to unblock the CI. The function should be fixed so that the disable is no longer needed.

**Comment by djyou** [comment]
Are you looking for a general cleanup or just a fix in this file? I noticed there are many occurrences of this disable being used. https://github.com/Azure/azure-cli/search?q=inconsistent-return-statements&type=Code&utf8=%E2%9C%93

I also don't totally agree with this pattern. Even a strongly-typed language would allow either returning or throwing.

**Comment by tjprescott** [comment]
Yes, Pylint's complaints of inconsistent return statements are often nonsense.

**Comment by troydai** [comment]
This one surfaced after the pylint is upgraded to 1.8.2. That's why I created this issue. Please use your best judgement. If you do deem this is not an issue feel free to close this bug.

**Comment by troydai** [comment]
I took another look. The pylint error makes sense in this case. The issue is with the last branch:
https://github.com/Azure/azure-cli/blob/a2ec550db69eae478d19e3fe7a29150bc4fef8a0/src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/repository.py#L106
It doesn't throw and exit the condition statement. Then in the next test condition when the `errorMessage` is `None` or empty, the exception on line 109 (https://github.com/Azure/azure-cli/blob/a2ec550db69eae478d19e3fe7a29150bc4fef8a0/src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/repository.py#L109) is not thrown leave the function a statue with "no return".

Fix is simple, add a `return None` to the end of the method.
